tests/test_core.py

```diff
@@ -1103,13 +1104,20 @@ class CustomNonMultipleField(CustomMultiplexingField):
         args = {"foos": fields.List(fields.Str())}
         result = parser.parse(args, web_request, location="query")
         assert result["foos"] in (["a", "b"], ["b", "a"])
+    # case 5: the field is a Tuple (special case)
+    elif setting == "tuple_field":
+        # this should behave like the is_multiple=True case and produce a tuple
+        args = {"foos": fields.Tuple((fields.Str, fields.Str))}
+        result = parser.parse(args, web_request, location="query")
+        assert result["foos"] in (("a", "b"), ("b", "a"))
+    # case 6: the field is custom, but added to the known fields of the parser
     elif setting == "added_to_known":
         # if it's included in the known multifields and is_multiple is not set, behave
         # like is_multiple=True
         parser.KNOWN_MULTI_FIELDS.append(CustomMultiplexingField)
         args = {"foos": CustomMultiplexingField()}
         result = parser.parse(args, web_request, location="query")
-        assert result["foos"] in ("a", "b")
+        assert result["foos"] in (["a", "b"], ["b", "a"])
```

Comment:
I'm wondering why it passed before the change, how this change is related to the `Tuple` change.

Reply:
Ah, sorry, I forgot to mention this in the PR. It's not related to the Tuple change but is a fix for this test (which was broken prior to this PR).

I switched a line in the test parser:
```diff
-        return MultiDictProxy(req.query, schema)
+        return self._makeproxy(req.query, schema)
```

This test was actually incorrect before (!), but because I copied the wrong `assert` from one of the other cases and didn't fix the test parser, I didn't notice it when I added it.

Since `CustomMultiplexingField` is being added to the parser as a multi-field, it should return a list in this case.